### ISSUE TITLE
Allow file upload (video, pdf, images) in E2E

### ISSUE THREAD

**Issue by phpgeekfr** [blockframes/blockframes#8900]
Allow file upload (video, pdf, images) in E2E.

Maybe by updating `blockframes-ci` storage rules ?

**Comment by phpgeekfr** [comment]
- [ ] unit tests to test upload to jwPlayer ( get videoUrl already covered by `apps/backend-functions/src/player.spec.ts`) ?
- [ ] same for imgix with public or protected media

**Comment by phpgeekfr** [comment]
- [ ] put default image in `/public/default-image.webp` on prod (and on ci until the next anon db)

**Comment by phpgeekfr** [comment]
- [ ] update notion for new `npm start emulatorsE2E` command https://www.notion.so/cascade8/E2E-testing-with-Emulator-4069418c783444a3951247781040337c

**Comment by phpgeekfr** [comment]
- [ ] activated only on festival for now. edit `apps/catalog/catalog/src/environment/environment.emulator.ts` to active it for catalog

**Comment by phpgeekfr** [comment]
note that imgIx is blocked during e2e tests:
```
  "blockHosts": [
    "*google-analytics.com",
    "*googleapis.com",
    "*imgix.net"
  ],
```

**Comment by phpgeekfr** [comment]
> * [x]  put default image in `/public/default-image.webp` on prod (and on ci until the next anon db)
> 
>     * [x]  put default pdf in `/public/default-pdf.pdf` on prod (and on ci until the next anon db)

Useless for now because imgIx is blocked by cypress

**Comment by phpgeekfr** [comment]
- Added a new unit-test test to `apps/backend-functions/src/player.spec.ts` that creates a file on storage and uploads it to jwPlayer and checks upload status.
- Existing unit-tests are already checking imgIx in `libs/media/src/lib/service.spec.ts`
- Updated existing E2E tests to upload a screener and a profile picture and check that db have correct values.

**Comment by phpgeekfr** [comment]
- [ ] using `getStorageEmulator` create a new unit test that checks that file is moved from /tmp to correct directory ?

**Comment by phpgeekfr** [comment]
- [ ] debug unit tests that are green even if tests failed and update https://www.notion.so/cascade8/Unit-testing-setup-710c09dfbc8f4119bbf0e8a0be7ba6bf

**Comment by phpgeekfr** [comment]
> using getStorageEmulator create a new unit test that checks that file is moved from /tmp to correct directory ?

Covered in e2e directly

**Comment by phpgeekfr** [comment]
> Added a new unit-test test to `apps/backend-functions/src/player.spec.ts` that creates a file on storage and uploads it to jwPlayer and checks upload status

not working well, removed
